libfabric.spec

```
@@ -82,7 +80,7 @@ Requires: libpsm2-devel >= 11.2.78
 Development files for the libfabric library.
 
 %prep
-%autosetup %autosetup_args
+%autosetup -n libfabric-%dl_version
```

Then make a portable `%{autosetup}` that doesn't need changing every time:
```suggestion
%autosetup -p1 -n libfabric-%dl_version
```